website/util/rubeus.py

```diff
@@ -321,7 +321,7 @@ def _serialize_node(self, node, visited=None, parent_is_folder=False):
             to_expand = False
 
         return {
-            'name': node.title if can_view else u'Private Component',
+            'name': node.title.replace('&amp;', '&') if can_view else u'Private Component',
```

Comment:
Can you add a `TODO` comment to remove this once the HTML-safe changes are made?